tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+requires =
+    tox>=4.2
+env_list =
+    py311
+    py310
+    py38
+
+[testenv]
+commands =
+    pytest --cov betterproto
+poetry_dep_groups =
+    dev
+require_locked_deps = true
+require_poetry = true
```

Comment:
I don't have a problem with keeping `isolated_build`, but I'm not sure what the benefit is of specifying the requirement inside `tox.ini` rather than `pyproject.toml`. The `tox-poetry-installer` docs recommend against it, but I'm not sure what the behaviour difference would be. Having it in `pyproject.toml` has caught the fact that the project requires `python>=3.7` but `tox-poetry-installer` doesn't support 3.7, which having it in `tox.ini` seemed to not flag.

Reply:
It appears that `isolated_build` is no longer an option, as the behaviour is now the default. https://tox.wiki/en/latest/upgrading.html#isolated-environment-on-by-default